server.js

```diff
@@ -1,10 +1,12 @@
 const Discord = require("discord.js");
-const colors = require('colors');
 const request = require("request");
 const log = require("log4js").getLogger('bot');
+const _ = require("lodash");
+require('colors');
+
 log.setLevel(process.env.LOG_LEVEL || "INFO");
 
-const commandChar = process.env.COMMAND_CHAR || "!";
+const commandChar = process.env.COMMAND_CHAR || "!"; // single character
```

Comment:
Well, it *is* possible to use multiple characters, although this may cause some unexpected fringe-case behaviors with the RegExp pattern. I clarified the comment to *encourage* single characters.